modelator/pytest/decorators.py

```diff
@@ -10,11 +10,33 @@
 from typing import Callable
 
 import pytest
+from munch import munchify
+from pyrsistent import freeze
 from pytest import FixtureRequest
 
 from modelator.Model import Model
 
 
+def sanitize_itf(obj):
+    match obj:
+        case dict():
+            if "#map" in obj:
+                return {
+                    freeze(sanitize_itf(k)): sanitize_itf(v) for (k, v) in obj["#map"]
+                }
+            if "#set" in obj:
+                return {freeze(sanitize_itf(e)) for e in obj["#set"]}
```

Comment:
`freeze` is needed, because TLA+ allows maps, sets, and lists as function keys or set elements, which are natively unhashable in Python.